libc/vga/vga.internal.h

```diff
@@ -0,0 +1,118 @@
+#ifndef COSMOPOLITAN_LIBC_VGA_VGA_INTERNAL_H_
+#define COSMOPOLITAN_LIBC_VGA_VGA_INTERNAL_H_
+
+/*
+ * VGA_TTY_HEIGHT, VGA_TTY_WIDTH, & VGA_USE_WCS are configuration knobs
+ * which can potentially be used to tweak the features to be compiled into
+ * our VGA teletypewriter support.
+ */
+
+/**
+ * Height of the video screen, in character units.  Undefine if the height
+ * may vary at runtime.
+ */
+#define VGA_TTY_HEIGHT 25
+/**
+ * Width of the video screen, in character units.  Undefine if the width may
+ * vary at runtime.
+ */
+#define VGA_TTY_WIDTH 80
+/**
+ * If VGA_USE_WCS is defined, the tty code can maintain an array of the
+ * Unicode characters "underlying" the 8-bit (or 9-bit) characters that are
+ * actually displayed on the text screen.  This can be used to implement
+ * something similar to Linux's /dev/vcsu* facility.
+ *
+ * @see lkml.kernel.org/lkml/204888.1529277815@turing-police.cc.vt.edu/T/
+ */
+#undef VGA_USE_WCS
```

Comment:
is this correct, its check for VGA_USE_WCS later down the header, and it will turn it off for `writev-vga.c`

Reply:
Hello @paulwratt,

Yes.  This is intentional.
```
/*
 * VGA_TTY_HEIGHT, VGA_TTY_WIDTH, & VGA_USE_WCS are configuration knobs
 * which can potentially be used to tweak the features to be compiled into
 * our VGA teletypewriter support.
 */
```
Thank you!